### DISCUSSION TITLE
training only on forces data

### DISCUSSION THREAD

**Discussion by benshpiro** [deepmodeling/deepmd-kit#2326] [Q&A] [answered]
I would like to train only on forces (and not use energies or the virial). What should I add to the input.json to do this? 
What should be the name of the type of loss function?

**Comment by Yi-FanLi** [comment, marked as answer]
Hi, I think you can directly put force data in your training path, without energy file.  If you also want to change the loss function, you can set "start_pref_e": 0.00 and "limit_pref_e": 0.00 in the input.json file. But I guess it would still be OK if you just leave them unchanged. DeePMD-kit will detect which kind of data is in the training data directory and use the existing data.

**Reply by benshpiro** [reply]
what about the type of loss function, I am guessing it shouldn't remain as "ener", no?

**Reply by Yi-FanLi** [reply]
It should remain as "ener". The other option is "tensor", which is designed for learning dipole or other tensors. "ener" learns from energy, force, and virial, while some of the three can be missing.

**Reply by benshpiro** [reply]
Thank you!!
